cli/src/utils.rs

```diff
@@ -209,28 +209,27 @@ pub async fn load_and_unlock_device(
                 }
             })?;
 
-            let access = DeviceAccessStrategy::Password {
+            DeviceAccessStrategy::Password {
                 key_file: device.key_file_path.clone(),
                 password,
-            };
-
-            // This will fail if the password is invalid, but also if the binary is compiled with fast crypto (see  libparsec_crypto)
-            libparsec::load_device(config_dir, &access).await?
-        }
-        DeviceFileType::Smartcard => {
-            let access = DeviceAccessStrategy::Smartcard {
-                key_file: device.key_file_path.clone(),
-            };
-
-            libparsec::load_device(config_dir, &access).await?
+            }
         }
-        DeviceFileType::Recovery | DeviceFileType::Keyring => {
+        DeviceFileType::Smartcard => DeviceAccessStrategy::Smartcard {
+            key_file: device.key_file_path.clone(),
+        },
+        DeviceFileType::Keyring => DeviceAccessStrategy::Keyring {
+            key_file: device.key_file_path.clone(),
+        },
+        DeviceFileType::Recovery => {
             return Err(LoadAndUnlockDeviceError::UnsupportedAuthentication(
                 device.ty,
             ));
         }
     };
 
+    // This will fail if the password is invalid, but also if the binary is compiled with fast crypto (see  libparsec_crypto)
```

Comment:
You can remove this comment since fast crypto in `libparsec_crypto` has been removed (see https://github.com/Scille/parsec-cloud/pull/6699)

Reply:
done